core/src/main/scala/dagr/core/cmdline/parsing/DagrCommandLineParser.scala

```diff
@@ -263,15 +263,15 @@ class DagrCommandLineParser(val commandLineName: String, val includeHidden: Bool
       entries
         .sortWith((lhs, rhs) => lhs.getSimpleName.compareTo(rhs.getSimpleName) < 0)
         .foreach{clazz =>
-          val property: CLPAnnotation = tasksToProperty.get(clazz).get
-          if (null == property) {
+          val clpAnnotation: CLPAnnotation = tasksToProperty.get(clazz).get
+          if (null == clpAnnotation) {
             throw new BadAnnotationException(s"Unexpected error: did not find the CommandLineTaskProperties annotation for '${clazz.getSimpleName}'")
           }
           if (clazz.getSimpleName.length >= 45) {
-            builder.append(wrapString(KGRN, String.format(s"    %s    %s\n", clazz.getSimpleName, wrapString(KCYN, property.description)), KNRM))
+            builder.append(wrapString(KGRN, String.format(s"    %s    %s\n", clazz.getSimpleName, wrapString(KCYN, formatShortDescription(clpAnnotation.description))), KNRM))
```

Comment:
I feel like this line is probably unreachable - i.e. if we've hit this point it's because the user managed to select a class, which implies it had `@CLP` on it.  However, if this is reachable, it'll NPE because we're in a branch where `null == clpAnnotation` and then it calls `clpAnnotation.description`.